SharedSources/VLCMediaLibraryManager.swift

```diff
@@ -219,6 +228,26 @@ extension VLCMediaLibraryManager {
     }
 }
 
+extension VLCMediaLibraryManager: VLCMediaFileDiscovererDelegate {
+    func mediaFileAdded(_ filePath: String!, loading isLoading: Bool) {
+        guard !isLoading else {
+            return
+        }
+        /* exclude media files from backup (QA1719) */
+        var excludeURL = URL(fileURLWithPath: filePath)
+        var resourceValue = URLResourceValues()
+
+        resourceValue.isExcludedFromBackup = true
+        try? excludeURL.setResourceValues(resourceValue)
+
+        reload()
+    }
+
+    func mediaFileDeleted(_ filePath: String!) {
+        reload()
```

Comment:
We should not have an entire reload for mediaFileDeleted. Instead we should just remove from the collectionView the file that belongs to that path

Reply:
Here, I'm reloading the medialibrary not the UI.
UI related updates(addition/deletion by indexes) should be addressed in another PR in the respective models.